test/e2e/suite/trustpolicy/trust_store.go

```diff
@@ -22,12 +22,12 @@ var _ = Describe("notation trust policy trust store test", func() {
 	})
 
 	It("invalid trust store", func() {
-		Host(BaseOptions(), func(notation *utils.ExecOpts, _ *Artifact, vhost *utils.VirtualHost) {
+		Host(BaseOptionsWithExperimental(), func(notation *utils.ExecOpts, _ *Artifact, vhost *utils.VirtualHost) {
```

Comment:
Should we create another set of test without experimental flags?

Reply:
We should, but IMO not in this PR. Our e2e `testdata` is still using OCI artifact manifest and referrers API as default. Changes of e2e test cases in this PR are only making the existing tests runnable again. We need another PR, maybe after rc.5, to update our e2e framework (there's a certain amount of work regarding this).

Reply:
> Should we create another set of test without experimental flags?

We should, but IMO not in this PR. Our e2e `testdata` is still using OCI artifact manifest and referrers API as default. Changes of e2e test cases in this PR are only making the existing tests runnable again. We need another PR, maybe after rc.5, to update our e2e framework (there's a certain amount of work regarding this). /cc: @JeyJeyGao

Reply:
I created a new issue to track this: https://github.com/notaryproject/notation/issues/668